src/main/kotlin/org/jitsi/jibri/capture/ffmpeg/FfmpegCapturer.kt

```diff
@@ -83,14 +85,14 @@ class FfmpegCapturer(
 
     init {
         val osType = osDetector.getOsType()
-        logger.debug("Detected os as OS: $osType")
+        logger.debug { "Detected os as OS: $osType" }
         getCommand = when (osType) {
             OsType.MAC -> { sink: Sink -> getFfmpegCommandMac(FfmpegExecutorParams(), sink) }
             OsType.LINUX -> { sink: Sink -> getFfmpegCommandLinux(FfmpegExecutorParams(), sink) }
             else -> throw UnsupportedOsException()
         }
 
-        ffmpeg.addStatusHandler(this::onFfmpegProcessUpdate)
+        this.ffmpeg.addStatusHandler(this::onFfmpegProcessUpdate)
```

Comment:
The `this` here (and a few other places) is necessary because otherwise `ffmpeg` resolves to the one passed in to the constructor, which is nullable.  The other alternative is to make the ctor arg something like `_ffmpeg`, but I find that awkward when calling using named parameters, so went with this approach.